### PR TITLE
build x64 release builds on appveyor

### PR DESCRIPTION
**Pull request by minrk**
x86 builds don't seem to work, nor do debug builds. Not sure why either fails.

Build completes now, but there appears to be a crash when trying to send messages.
At least the DLL gets loaded.

### CODE REVIEW

**Comment by minrk** [comment]
debug build is failing when I call `socket.send('test')`:

screen shot 2016-08-23 at 14 51 21

I'm really not sure how to deal with that, since I know ~nothing about how the buffer stuff works on Windows. It seems like this is a real bug in the binding at least, but I can't be 100% sure.

**Comment by rgbkrk** [comment]
We did swap the Windows DLLs out at some point here for ones that either I or @jdfreder built. If we need to nuke those we can. Want to merge this as is to at least up date the AppVeyor setup?

**Comment by minrk** [comment]
I've just swapped out the DLLs (and binding.cc, index.js) for the upstream ones and it's working now (still not x86), so the next appveyor build *should* go green.

**Comment by minrk** [comment]
Appveyor is green! I'm not sure how important 32b Windows builds are, but at least *something* is working for now. As far as nteract is concerned, at least, I would hope that 64b is sufficient.

**Comment by rgbkrk** [comment]
Wooooooooooo

**Comment by rgbkrk** [comment]
One test on Travis is timing out (and retrying it looks like). I'll go ahead and merge as this is moving forward for appveyor & windows! Thank you Min!

**Comment by rgbkrk** [comment]
> I would hope that 64b is sufficient

Yeah I think it's reasonable for us to only support modern Windows. Windows Server 2008 was the last 32-bit OS released by Microsoft for the server. For the Desktop, I think it was Windows 7, which released in 2009.

File: binding.cc [hunk 1]
```diff
@@ -1382,7 +1611,7 @@ extern "C" NAN_MODULE_INIT(init) {
       _wfullpath(path, pathDir, MAX_PATH);
       set_dll_directory(path);
       caller.set_func(set_dll_directory);
-      assert (!FAILED(__HrLoadAllImportsForDll("libzmq.dll")) &&
+      assert (!FAILED(__HrLoadAllImportsForDll("libzmq-v100-mt-4_0_4.dll")) &&
```
Comment (rgbkrk) [inline binding.cc]: Does this mean we'll have to package the DLL directly?

**Pull request merged**